build_projects/dotnet-cli-build/TestTargets.cs

```diff
@@ -14,43 +14,43 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class TestTargets : Task
     {
-        private static string s_testPackageBuildVersionSuffix = "<buildversion>";
-
-        public static readonly string[] TestProjects = new[]
-        {
-            "ArgumentForwardingTests",
-            "crossgen.Tests",
-            "EndToEnd",
-            "dotnet.Tests",
-            "dotnet-build.Tests",
-            "dotnet-build3.Tests",
-            "dotnet-compile.Tests",
-            "dotnet-compile.UnitTests",
-            // TODO: https://github.com/dotnet/cli/issues/3558
-            // "dotnet-compile-fsc.Tests",
-            "dotnet-new.Tests",
-            "dotnet-pack.Tests",
-            "dotnet-publish.Tests",
-            "dotnet-resgen.Tests",
-            "dotnet-run.Tests",
-            "dotnet-run.UnitTests",
-            "dotnet-test.Tests",
-            "dotnet-test.UnitTests",
-            // TODO: https://github.com/dotnet/cli/issues/3216
-            //"Kestrel.Tests",
-            "Microsoft.DotNet.Cli.Utils.Tests",
-            "Microsoft.DotNet.Compiler.Common.Tests",
-            "Microsoft.DotNet.ProjectModel.Tests",
-            "Microsoft.DotNet.ProjectModel.Loader.Tests",
-            "Microsoft.Extensions.DependencyModel.Tests",
-            "Microsoft.DotNet.Configurer.UnitTests",
-            "Performance"
-        };
-
-        public static readonly string[] WindowsTestProjects = new[]
-        {
-            "binding-redirects.Tests"
-        };
+        //private static string s_testPackageBuildVersionSuffix = "<buildversion>";
```

Comment:
We can't comment out/remove these yet until we switch our Jenkins and official builds to use MSBuild.